pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -109,7 +116,9 @@ func TestCMServerDefaults(t *testing.T) {
 			ClusterName:                       "kubernetes",
 			TerminatedPodGCThreshold:          12500,
 			VolumeConfiguration: componentconfig.VolumeConfiguration{
+				EnableDynamicProvisioning:  true,
 				EnableHostPathProvisioning: false,
+				FlexVolumePluginDir:        "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
```

Comment:
I don't know what this is/does.  Do you know? If not, issue to ask @markturansky if this is a knob we need to let people touch.